src/services/mongoose/accounts.ts

```diff
@@ -226,4 +229,5 @@ const translateToAccount = (result: AccountRecord): Account => ({
       }),
     ) || [],
   kratosUserId: result.kratosUserId as UserId,
+  displayCurrency: result.displayCurrency || DisplayCurrency.Usd,
```

Comment:
not sure about the syntax

```suggestion
  displayCurrency: result.displayCurrency as DisplayCurrency || DisplayCurrency.Usd,
```

Reply:
currently we have a DisplayCurrency type (enum), we will replace it in the next PR because we cant use the same because currencies are dynamic